app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -27,17 +28,29 @@ def idme
       return redirect_to params["after_login"]
     end
 
-    should_delete_duplicate_intake = session[:intake_id] && (is_returning_user || is_new_spouse)
+    # A new intake is created before sign in so that we can save answers to questions that are asked before sign in.
+    # If a returning user signs in, we want to delete this new intake because the user will already have an intake.
+    # TODO: We also want to do this for a new spouse - why??
```

Comment:
can someone remind me why we delete duplicate intake for spouse? why is there a duplicate intake when the spouse is signing in?